source/_docs/blueprint/schema.markdown

```diff
@@ -140,6 +146,70 @@ It is then up to the user to find out what to enter there. Blueprints that come
 
 A blueprint can have as many inputs as you like.
 
+### Blueprint input sections
+
+One or more input sections can be added under the main `input` key. Each section visually groups the inputs in that section, 
+allows an optional description, and optionally allows for collapsing those inputs. 
+
+A section is differentiated from an input by the presence of an additional `input` key within that section. 
+
+<div class='note warning'>
+Input sections are a new feature in version 20??.?.?. Set the `min_version` for the blueprint to at least this version if using input sections, otherwise the blueprint will generate errors on older versions. 
```

Comment:
```suggestion
Input sections are a new feature in version 2024.6. Set the `min_version` for the blueprint to at least this version if using input sections. Otherwise, the blueprint will generate errors on older versions. 
```